starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -42,23 +42,33 @@ public List<Inventory> findInventories() {
 
   /**
    * Create an Inventory.
-   * @param inventory the inventory to be created.
+   * @param inventory The inventory to be created.
    * @return The created inventory.
    */
   @PostMapping
-  public Inventory createInventory(@RequestBody Inventory inventory){
+  public Inventory createInventory(@RequestBody Inventory inventory) {
     return this.inventoryDAO.create(inventory);
   }
 
+  /*
+   * Update an Inventory.
+   * @param inventory The inventory to be updated.
+   * @return The updated inventory.
+   */
+  @PostMapping("/update")
+  public Inventory updateInventory(@RequestBody Inventory inventory) { 
+    return this.inventoryDAO.updateMe(inventory);
```

Comment:
this method will be similar to your deleteInventory method that you have where you get an optional back from this.inventoryDAO.update(inventory). You will want to check to see if its empty. If so return null other wise return the saved inventory